assembly/utils/Arrays.ts

```diff
@@ -19,4 +19,16 @@ export namespace ArrayUtils {
         }
         return true;
     }
+
+    /**
+     * Returns new Array<u8> from Uint8 Typed array
+     * @param typedArr 
+     */
+    export function toU8Array(typedArr: Uint8Array): u8[] {
+        let res = new Array<u8>(2);
```

Comment:
why are you creating this array with capacity 2? What if `typedArr` has length > than 2?